tools/mtf

```diff
@@ -24,7 +25,7 @@ while [[ $# -gt 0 ]]; do
         exit 0
       ;;
       -l|--linters)
-        AVOCADO_ARGS="$AVOCADO_ARGS $SITE_LIB/$MTF_TOOLS/*.py"
+        AVOCADO_ARGS="$AVOCADO_ARGS $MTF_TOOLS/modulelint/*.py"
```

Comment:
This is wrong. All modulelint are mentioned in site lib. 
We move `/usr/share/moduleframework/tools` to `/usr/lib/python2.7/site-packages/moduleframework/tools`.

Reply:
We've dicsussed it. It is fixed in devel branch and repaired previous code SITE_LIB code what didn't work.
Actually merge has to be done carefully manually. (it is in conflict with devel) or rebased here.

Reply:
done